test/runtests.jl

```diff
@@ -13,3 +13,16 @@ using Test
 
     @test all(P[101, :] .== 1)
 end
+
+using Images: shepp_logan
+using Statistics
+@testset "iradon - shepp logan" begin
+    Igt = shepp_logan(128)
+    views = 200
+    θ = range(0, 2π, length=views)
+    t = -150:150
+    P = radon(Igt, θ, t)
+    I = iradon(P, θ, t)
+    diff = I .- Igt
+    @test mean(diff[:]) ≈ 0 atol=1e-3
```

Comment:
This may not be the most sensitive test, would it be better to test `sad`?

Reply:
Yes, you're right. `sad` is a better metric.